arch/xtensa/src/esp32/esp32_wireless.c

```diff
@@ -288,6 +370,476 @@ void esp32_phy_disable_clock(void)
   leave_critical_section(flags);
 }
 
+#ifdef CONFIG_ESP32_SUPPORT_MULTIPLE_PHY_INIT_DATA
+
+/****************************************************************************
+ * Name: phy_crc_check
+ *
+ * Description:
+ *   Check the checksum value of data
+ *
+ * Input Parameters:
+ *   data     - Data buffer pointer
+ *   length   - Data length
+ *   checksum - Checksum pointer
+ *
+ * Returned Value:
+ *   OK on success; a negated errno on failure
+ *
+ ****************************************************************************/
+
+static int phy_crc_check(uint8_t *data, const uint8_t *checksum,
+                         size_t length)
+{
+  uint32_t crc_data = 0;
+  crc_data = crc32_le(crc_data, data, length);
+  uint32_t crc_size_conversion = htonl(crc_data);
+  if (crc_size_conversion != *(uint32_t *)(checksum))
+    {
+      return ERROR;
+    }
+
+  return OK;
+}
+
+/****************************************************************************
+ * Name: phy_find_bin_type_according_country
+ *
+ * Description:
+ *   Find the PHY initialization data type according to country code
+ *
+ * Input Parameters:
+ *   country - Country code pointer
+ *
+ * Returned Value:
+ *   PHY initialization data type
+ *
+ ****************************************************************************/
+
+static uint8_t phy_find_bin_type_according_country(const char *country)
+{
+  uint8_t i;
+  uint8_t phy_init_data_type;
+  uint8_t num = sizeof(g_country_code_map_type_table) /
+                sizeof(phy_country_to_bin_type_t);
+  for (i = 0; i < num; i++)
+    {
+      if (!memcmp(country, g_country_code_map_type_table[i].cc,
+          sizeof(g_phy_current_country)))
```

Comment:
```suggestion
      if (!memcmp(country, g_country_code_map_type_table[i].cc,
                  sizeof(g_phy_current_country)))
```
Why use `sizeof(g_phy_current_country)` and not `sizeof(g_country_code_map_type_table[i].cc)` or `PHY_COUNTRY_CODE_LEN`?

Reply:
Done, already using `PHY_COUNTRY_CODE_LEN`